newIDE/app/src/InstancesEditor/Grid.js

```diff
@@ -25,10 +38,14 @@ export default class SelectionRectangle {
       return;
     }
 
+    const gridColor = options.gridColor || DEFAULT_COLOR;
+    const gridHex = rgbColorToHex(gridColor);
+    const gridAlpha = gridColor.a || 1;
```

Comment:
Seems like the alpha is not selectable, we might keep it always at 1?

Reply:
Alpha is configurable with the color picker, and it works as expected here. The fallback of 1 comes up because `a` is an optional property, or if the user selects `0` which would be confusing to have a visible grid that is invisible.